lib/datasource/index.ts

```diff
@@ -214,7 +214,7 @@ export function getDefaultVersioning(datasourceName: string): string {
 }
 
 function applyReplacements(
-  config: GetReleasesInternalConfig
+  config: GetPkgReleasesConfig
```

Comment:
Unrelated change?

Reply:
Yep 😅 I was investigating replacements in the same branch.